src/store/mutations.js

```diff
@@ -52,58 +52,55 @@ export async function getIndexPageData() {
     graphEvents = graphEvents.concat(xdaiEvents)
   }
 
-    let mr = {}
-    let up = {}
-    let mC = {}
-    let hPP = {}
-
-    if(poapEvents && poapEvents.length) {
-      mr = poapEvents[0]
-      up = poapEvents[0]
-      mC = poapEvents[0]
-      hPP = poapEvents[0]
-    }
-    let isMostRecent = false
-
-    for (let i = 0; i < poapEvents.length; i++) {
-      const ev = poapEvents[i];
-      ev.tokenCount = 0
-      ev.transferCount = 0
-      for (let j = 0; j < graphEvents.length; j++) {
-        const gev = graphEvents[j];
-        if(ev.id === parseInt(gev.id)) {
-          ev.tokenCount += parseInt(gev.tokenCount)
-          ev.transferCount += parseInt(gev.transferCount)
-        }
-      }
-      let now = new Date().getTime()
-      let evDate = new Date(ev.start_date.replace(/-/g, ' ')).getTime()
-
-      if(evDate > now) {
-        up = ev
+  let mr, up, mC, hPP;
+
+  let isMostRecent = false
+
+  for (let i = 0; i < poapEvents.length; i++) {
+    const ev = poapEvents[i];
+    ev.tokenCount = 0
+    ev.transferCount = 0
+    for (let j = 0; j < graphEvents.length; j++) {
+      const gev = graphEvents[j];
+      if(ev.id === parseInt(gev.id)) {
+        ev.tokenCount += parseInt(gev.tokenCount)
+        ev.transferCount += parseInt(gev.transferCount)
       }
+    }
+    let now = new Date().getTime()
+    let evDate = new Date(ev.start_date.replace(/-/g, ' ')).getTime()
```

Comment:
`evDate` is event date?

Reply:
yes, I was keeping consistent with old naming (ev is an event variable)